autoinstallation/README.md

```diff
@@ -84,6 +84,17 @@ local findBiggestDisk(disks) =
   local sizedDisks = std.filter(function(d) std.objectHas(d, 'size'), disks);
   local sorted = std.sort(sizedDisks, function(x) x.size);
   sorted[0].logicalname;
+local selectClass(lshw, class) =
```

Comment:
IMHO, we should offer this as some kind of helper. I would not expect the users to have to include this on their own profiles.

Reply:
yeap, that is open question. Basically I think we should provide something like jsonnet library that is also injected into jsonnet during evaluation. At least two helpers will be useful..`selectClass` for class filtering and `findID` to find one specific id ( useful to e.g. check physical memory ).